webapp/locales/en.json

```diff
@@ -423,7 +423,8 @@
     "comment": {
       "submit": "Comment",
       "submitted": "Comment Submitted",
-      "updated": "Changes Saved"
+      "updated": "Changes Saved",
+      "answered": "answered"
```

Comment:
I would name this `"answer": "Answer"`.
It should be the present form, because you will do it right now – answer it.
Answered is the past form.

Reply:
ok